plugins/Login/Login.php

```diff
@@ -41,6 +42,12 @@ public function getJsFiles(&$jsFiles)
         $jsFiles[] = "plugins/Login/javascripts/login.js";
     }
 
+   public function getStylesheetFiles(&$stylesheetFiles)
+    {
+        $stylesheetFiles[] = "plugins/Login/stylesheets/login.less";
+        $stylesheetFiles[] = "plugins/Login/stylesheets/variables.less";
```

Comment:
Did you forget to add the file 'variables.less'. Doesn't exist in current master. I'll remove that line. Please readd it if that file was just missing.